packages/react/src/createDOMRenderer.test.tsx

```diff
@@ -13,11 +14,11 @@ describe('createDOMRenderer', () => {
 
     const clientRenderer = createDOMRenderer(document);
     const serverRenderer = createDOMRenderer(
-      // we should use "null" as "undefined" will fallback to "document" which is present in this environment
+      // we should use "null" as "undefined" will fall back to "document" which is present in this environment
       null as unknown as undefined,
```

Comment:
nit: perhaps we should update `createDOMRenderer` types to support this scenario also?

Reply:
Not really, as it expected to have `document` on client side always.